dev/com.ibm.ws.jaxrs.2.1_fat_extended/fat/src/com/ibm/ws/jaxrs21/fat/extended/PackageJsonBTestWithFeature.java

```diff
@@ -24,12 +24,13 @@
 import componenttest.annotation.SkipForRepeat;
 import componenttest.annotation.TestServlet;
 import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.JakartaEE10Action;
 import componenttest.rules.repeater.JakartaEE9Action;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.FATServletClient;
 import jaxrs21.fat.jsonb.JsonBTestServlet;
 
-@SkipForRepeat(JakartaEE9Action.ID) //JSON-B Provider implementation seems to be broken in Jakarta package space...
+@SkipForRepeat({JakartaEE9Action.ID, JakartaEE10Action.ID}) //JSON-B Provider implementation seems to be broken in Jakarta package space...
```

Comment:
should open an investigation issue for this

Reply:
https://github.com/OpenLiberty/open-liberty/issues/22521